app/views/events/show.html.erb

```diff
@@ -0,0 +1,26 @@
+   <div class="row">
+     <div class="col-md-6 ">
+       <h2>Attendees for event:</h2>
+       <h4>" <%= @event.name %> "</h4><br>
+       <ul>
+          <% @event.users.each do |users| %>
+          <li><b><p><%= users.name %></p></b></li>
+           <% end %>
+       </ul>
+      </div>
+      <% if current_user.id == @event.creator_id %>
```

Comment:
I'd like the application to do this check somewhere else so we can avoid having logic in the views